functions/FileUploadedFunc/__init__.py

```diff
@@ -33,11 +33,10 @@ def main(myblob: func.InputStream):
     try:
         time.sleep(random.randint(1, 2))  # add a random delay
         statusLog = StatusLog(cosmosdb_url, cosmosdb_key, cosmosdb_log_database_name, cosmosdb_log_container_name)
-        statusLog.upsert_document(myblob.name, 'Pipeline triggered by Blob Upload', StatusClassification.INFO, State.PROCESSING, False)            
+        statusLog.upsert_document(myblob.name, 'Pipeline triggered by Blob Upload', StatusClassification.INFO, State.PROCESSING, True)            

```

Comment:
This was set to false so it doesn't overwrite the "Uploaded" status from the UX. With this set to True then we won't know the timestamp for when a file was uploaded from the UX. If you re-upload a file from the UX it has this set to True to start a new log. 
I suggest we keep this to "False" here.

Reply:
reverted